lib/awful/widget/tasklist.lua

```diff
@@ -24,6 +24,50 @@
 -- <tr><td>⬍</td><td><a href="./client.html#client.maximized_vertical">maximized_vertical</a></td></tr>
 -- </table>
 --
+-- **Customizing the tasklist:**
+--
+-- The `tasklist` created by `rc.lua` use the default values for almost
+-- everything. However, it is possible to override each aspects to create a
+-- very different widget. Here's an example that create a tasklist similar to
+-- the default one, but with an explicit layout and some spacing widgets:
+--
+--@DOC_wibox_awidget_tasklist_rounded_EXAMPLE@
```

Comment:
Shouldn't these examples be added in the same commit that refers to them? Otherwise, the commit is not really self-contained.

Reply:
It doesn't change anything since the macro expands to nothing (thus ldoc works). I mostly ignored this type of "issue" in each of the PRs I submitted, I don't think it affects anyone and makes the commits overall smaller.

I would have to add the documentation in the first commit (otherwise I add undocumented features) and then edit the description in the second to refer to the example, modifying the same line in 2 commits in a row. That sounds like a little bikeshedding compared to just doing it this way. It also makes rebasing harder.

Reply:
Ok, I'm convinced.